perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -65,5 +62,8 @@ chmod +x perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
 ./perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
 sudo umount gcs
 
+# Executing integration tests
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/... -p 1 --integrationTest -v --testbucket=gcsfuse-integration-test
```

Comment:
should also run e2e tests at the end in case of periodic tests as well ?

Reply:
Ideally it should be before fio test only as swetha mentioned earlier as if E2E tests will fail it will terminate only from that point.

Reply:
okay, if we get similar errors in periodic test then we can change.